scripts/game_structure/windows.py

```diff
@@ -670,6 +669,10 @@ def process_event(self, event):
                 game.all_screens["profile screen"].exit_screen()
                 game.all_screens["profile screen"].screen_switches()
                 self.kill()
+        elif event.type == pygame.KEYDOWN and event.key == pygame.K_ESCAPE:
```

Comment:
Also need to check that the user has keybinds enabled (I think it's `game.settings["keybinds"]` but do double-check that).